publish.ps1

```diff
@@ -14,8 +14,8 @@ function do-prepareSource($build_server = "", $build_number = "")
     Write-Output "Setting build #defines..."
     # $build_number = "";
     # $build_server = "";
-    $build_branch = (git rev-parse --abbrev-ref HEAD)
-    $build_commit_sha1 = (git rev-parse HEAD)
+    $build_branch = $Env:APPVEYOR_REPO_BRANCH
```

Comment:
This isn't just for APPVEYOR to use, so it should pass it as an argument if necessary.